daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -69,10 +67,10 @@ class DecodeV1Spec
     forEveryVersionSuchThat(_ => true)
 
   private[this] def forEveryVersionBefore[U](maxVersion: LV): (LV => U) => Unit =
-    forEveryVersionSuchThat(v => v precedes maxVersion)
+    forEveryVersionSuchThat(_ < maxVersion)
```

Comment:
Maybe we should inline this function and the one below given how simple it is to write the version constraints now. I feel like these functions only add complexity without any gain in the new situation.

Reply:
done.